commit 85a5bef06a23c8c2c7aa03834eacd952865b342a
Author: slfan1989 <55643692+slfan1989@users.noreply.github.com>
Date:   Wed Oct 8 09:46:41 2025 +0800

    YARN-11875. Fix build failure caused by color@5.0.2. (#8016) Contributed by Shilun Fan
    
    * YARN-11875. Fix build failure caused by color@5.0.2.
    
    Reviewed-by:  Steve Loughran <stevel@cloudera.com>
    Signed-off-by: Shilun Fan <slfan1989@apache.org>
    (cherry picked from commit d9c33088628d7146664ba97e4ef5f196b5f12d15)

diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-applications/hadoop-yarn-applications-catalog/hadoop-yarn-applications-catalog-webapp/package.json b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-applications/hadoop-yarn-applications-catalog/hadoop-yarn-applications-catalog-webapp/package.json
index 59cc3da179f..f5dd22bffcb 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-applications/hadoop-yarn-applications-catalog/hadoop-yarn-applications-catalog-webapp/package.json
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-applications/hadoop-yarn-applications-catalog/hadoop-yarn-applications-catalog-webapp/package.json
@@ -20,6 +20,7 @@
         "apidoc": "0.17.7"
     },
     "resolutions": {
+        "color": "3.1.3",
         "triple-beam": "1.3.0"
     },
     "scripts": {
